### DISCUSSION TITLE
Similar Products and inspiration

### DISCUSSION THREAD

**Discussion by WillieCubed** [acmutd/comet_planning#101] [Ideas]
Since what we're doing isn't entirely unique, I think we should take some time to collect products that are related to planning a student's college experience. These need not be necessarily related to course planning.

@allisonschiang I know you found a few already, so feel free to start the discussion!

**Comment by Shaurya0108** [comment]
https://www.mystudylife.com/ 
https://powerplanner.net/platforms
Pretty cool apps, both are quite similar too.

**Comment by ZzRanger** [comment]
UTD Degree Planner - https://utdegreeplanner.com/ 
MIT Degree Planner - https://courseroad.mit.edu/road/

The UTD one was made by two current UTD students. The Table View is pretty neat :)
The MIT one looks a bit dated tbh but still worth looking at for inspiration.

**Comment by allisonschiang** [comment]
UC Berkeley (Go california!) https://berkeleytime.com/about
MIT Course picker: https://picker.mit.edu/
CMU vid (need to be student to access) https://www.youtube.com/watch?v=iFC_PtBhEeQ
Cornell: https://courseplan.io/

**Comment by allisonschiang** [comment]
uwash: https://uwgrades.com/

**Reply by ZzRanger** [reply]
Got 403 when I tried to access it. Looks like they moved to DawgPath but you need UW net ID and password to access.

DawgPath: http://dawgpath.uw.edu/

Source: https://www.reddit.com/r/udub/comments/suh58v/so_when_did_uwgradescom_get_removed/

<img width="828" alt="Screen Shot 2022-03-04 at 11 48 16 PM" src="https://user-images.githubusercontent.com/47403443/156870103-e2300976-8d11-4ee7-b94e-66df478cfb39.png">

**Comment by jasonappah** [comment]
I really liked the team page on Berkeleytime - overall the page is really clean. The backgrounds of the team headshots add some nice colors and patterns to the page without being detrimental to the clean aesthetic of the page.

**Comment by Kar-8** [comment]
prepler: https://www.prepler.com/ 
The landing page for this seems alright, it might be useful to look at

**Comment by jasonappah** [comment]
The UTD schedule planner requires that you have multiple tabs open for degree requirements and schedule planning. Sometimes it's hard to find up to date information.

**Comment by aanos-m** [comment]
MIT's Degree Planner: https://courseroad.mit.edu/road/$defaultroad$

<img width="339" alt="Screen Shot 2022-09-20 at 4 33 48 PM" src="https://user-images.githubusercontent.com/106491552/191368913-52016eae-75ae-4b9d-8d06-87c25f63fe20.png">

I like the drop-down menu they have when adding classes. We can hopefully incorporate it when searching for classes and obviously change the UI lolz.

**Comment by ZzRanger** [comment]
Dartmouth D-Planner is so clean 😍 😍 

https://d-planner.com/
